sql/updates/world/6.x/2016_05_13_00_world.sql

```diff
@@ -0,0 +1,370 @@
+(@CGUID+176, 2914, 732, 1, 1, 0, -1504.755, 769.7535, 124.5964, 5.682231, 120, 5, 1), -- 2914 (Area: 5400) (possible waypoints or random movement)
+-- Baradin Crocolisk (47591)
+(@CGUID+177, 47591, 732, 1, 1, 0, -1099.271, 1227.937, 123.4584, 1.089231, 120, 15, 1), -- 47591 (Area: 0) (possible waypoints or random movement)
+(@CGUID+178, 47591, 732, 1, 1, 0, -1136.39, 1103.76, 118.8463, 0, 120, 15, 1), -- 47591 (Area: 0) (Auras: ) (possible waypoints or random movement)
+(@CGUID+179, 47591, 732, 1, 1, 0, -1144.989, 1202.163, 120.7522, 4.369532, 120, 15, 1), -- 47591 (Area: 0) (possible waypoints or random movement)
+(@CGUID+180, 47591, 732, 1, 1, 0, -1163.222, 1140.002, 121.7983, 0.2406246, 120, 15, 1), -- 47591 (Area: 0) (Auras: )
+(@CGUID+181, 47591, 732, 1, 1, 0, -1183.716, 1307.183, 119.6709, 4.973243, 120, 15, 1), -- 47591 (Area: 0) (possible waypoints or random movement)
+(@CGUID+182, 47591, 732, 1, 1, 0, -1183.25, 1232.32, 119.6923, 0, 120, 15, 1), -- 47591 (Area: 5399) (possible waypoints or random movement)
+-- (@CGUID+183, 47591, 732, 1, 1, 0, -1198.97, 1224.732, 118.9178, 2.886917, 120, 15, 1), -- duplicate
+-- (@CGUID+184, 47591, 732, 1, 1, 0, -1204.302, 1309.922, 122.1487, 0.407713, 120, 15, 1), -- duplicate
+(@CGUID+185, 47591, 732, 1, 1, 0, -1215.161, 1194.385, 120.847, 4.482553, 120, 15, 1), -- 47591 (Area: 0) (Auras: ) (possible waypoints or random movement)
+(@CGUID+186, 47591, 732, 1, 1, 0, -1223.21, 1257.42, 121.1063, 0, 120, 15, 1), -- 47591 (Area: 0) (possible waypoints or random movement)
+(@CGUID+187, 47591, 732, 1, 1, 0, -1233.366, 1143.409, 119.9837, 0.7349917, 120, 15, 1), -- 47591 (Area: 5399)
+(@CGUID+188, 47591, 732, 1, 1, 0, -1246.8, 1303.09, 119.2313, 0, 120, 15, 1), -- 47591 (Area: 0) (Auras: ) (possible waypoints or random movement)
+-- (@CGUID+189, 47591, 732, 1, 1, 0, -1264.328, 1168.115, 119.8141, 0.4148644, 120, 15, 1), -- duplicate
+(@CGUID+190, 47591, 732, 1, 1, 0, -1265.58, 1241.87, 121.5823, 0, 120, 15, 1), -- 47591 (Area: 0) (possible waypoints or random movement)
+(@CGUID+191, 47591, 732, 1, 1, 0, -1274.22, 1167.16, 120.8103, 0, 120, 15, 1), -- 47591 (Area: 0) (Auras: ) (possible waypoints or random movement)
+(@CGUID+192, 47591, 732, 1, 1, 0, -1300.003, 1110.957, 121.0026, 0.2005581, 120, 15, 1), -- 47591 (Area: 5399)
+-- (@CGUID+193, 47591, 732, 1, 1, 0, -1314.573, 1182.476, 119.0194, 4.270839, 120, 15, 1), -- duplicate
+(@CGUID+194, 47591, 732, 1, 1, 0, -1305.82, 1200.89, 119.9623, 0, 120, 15, 1), 
+(@CGUID+195, 47591, 732, 1, 1, 0, -1305.22, 1262.41, 119.4313, 0, 120, 15, 1), -- 47591 (Area: 0) (possible waypoints or random movement)
+-- (@CGUID+196, 47591, 732, 1, 1, 0, -1318.213, 1121.961, 119.4406, 1.919687, 120, 15, 1), -- duplicate
+-- Problim (47593) waypoints needed
+(@CGUID+196, 47593, 732, 1, 1, 0, -1344.739, 971.6679, 123.2955, 1.409069, 120, 5, 1); -- 47593 (Area: 0) (Auras: 88361 - 88361) (possible waypoints or random movement)
+
+SET @OGUID := 68000;
```

Comment:
move SETs for spawns to top to allow commiter to see it needs to be set on easy way.